examples/embedded_c/embedded_c.c

```diff
@@ -585,6 +587,7 @@ PostResponser(struct mg_connection *conn, void *cbdata)
 }
 
 
+#if !defined(NO_FILESYSTEMS)
```

Comment:
This example can easily be modified to work without files.
1) Don't use `mg_modify_passwords_file` 
2) Replace `mg_check_digest_access_authentication` by ... probably I need a new function here